src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1349,12 +1349,8 @@ emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsi
 		if (!(!strcmp (m_class_get_name (cmethod->klass), "Vector128") || !strcmp (m_class_get_name (cmethod->klass), "Vector")))
 			return NULL;
 		switch (id) {
-		case SN_Create:
 		case SN_GetLower:
 		case SN_GetUpper:
-		case SN_Shuffle:
```

Comment:
These were removed as well, as `SN_Shuffle` hasn't been supported by LLVM either. `SN_ToVector128` and `SN_ToVector128Unsafe` were Vector64 API's and didn't apply to Vector128.